### DISCUSSION TITLE
How to make Repeater inside Repetear fields

### DISCUSSION THREAD

**Discussion by mialyGit** [laravel/nova-issues#5756] [Ideas & Feature Requests] [answered]
- Laravel Version: 9.52.12
- Nova Version: 4.26.5
- PHP Version: 8.1.0

### Description:

I have this code :

`Repeater::make('Line Items')
            ->fullWidth()
            ->repeatables([
                LineItem::make(),
            ])`

I want to add Repeater FullImageWidth inside LineItem :

```
    public function fields(NovaRequest $request)
    {
        return [
            Text::make('Quantity')->translatable()->fullWidth(),
            MediaHubField::make('Image', 'image')->translatable(),
            Repeater::make('FullWidthImage')->repeatables(
                [
                    FullImageWidth::make()
                ]
            )->asJson()
        ];
    }
```
This gives an error :
` Laravel\\Nova\\Fields\\Repeater\\Presets\\JSON::get(): Argument #2 ($model) must be of type Illuminate\\Database\\Eloquent\\Model, array given, called in C:\\Users\\USER\\Documents\\test_winehubpro\\apps\\nova\\vendor\\laravel\\nova\\src\\Fields\\Repeater.php on line 173`

Any has an idea ??

**Comment by davidhemphill** [comment, marked as answer]
Nested Repeaters are not currently supported.

**Reply by stephaneisenbach** [reply]
Would be a very useful feature to implement. ;-)

**Reply by nickpoulos** [reply]
This would be a very useful thing to note within the documentation. 

It could help prevent a situation where developers create a whole feature using Repeaters and then find out after the fact that they need to re-architect everything due to this restriction.

**Reply by frizzant** [reply]
How is one supposed to create dynamic content if this is not possible? This seems crucial to have.
